src/MoBi.Core/Services/SimulationConfigurationFactory.cs

```diff
@@ -37,11 +38,11 @@ public class SimulationConfigurationFactory : ISimulationConfigurationFactory
          _templateResolverTask = templateResolverTask;
       }
 
-      public SimulationConfiguration Create()
+      public SimulationConfiguration Create(SimulationSettings settings = null)
       {
          var simulationConfiguration = new SimulationConfiguration
          {
-            SimulationSettings = _cloneManager.Clone(_projectRetriever.Current.SimulationSettings)
+            SimulationSettings = settings ?? _cloneManager.Clone(_projectRetriever.Current.SimulationSettings)
```

Comment:
I am not sure this is right here. you are going to use the same references (but it comes from another simulation)
Won't you be corrupting it?

Reply:
Are there references to be worried about? Or do you mean that if we edit the properties of one simulation, we will change another as well?